src/plugins/HUDPlugin.js

```diff
@@ -1,69 +1,95 @@
-import { ButtonPlugin } from './ButtonPlugin';
-import { Button } from '../ui-elements';
+import { BasePlugin } from './BasePlugin';
+import { RadioGroup } from '../ui-elements';
 import { SavedData } from '../SavedData';
 
+const SUPPORTED_POSITIONS = ['top', 'bottom', 'left', 'right'];
+
 /**
  * @export
  * @class HUDPlugin
  * @extends {BasePlugin}
  */
-export class HUDPlugin extends ButtonPlugin {
+export class HUDPlugin extends BasePlugin {
   /**
    * Creates an instance of HUDPlugin
    * @param {object} params
-   * @param {string | HTMLElement} [params.hudSelectorButtons] string or HTML Element that represents the HTML button that toggles through the HUD positions
+   * @param {string | HTMLElement} [params.hudSelectorRadios] string that represents the HTML Radio Buttons that select the HUD Position
    * @memberof HUDPlugin
    */
-  constructor({ hudSelectorButtons } = {}) {
+  constructor(hudSelectorRadios, { defaultValue = 'top' } = {}) {
     super('HUD-Layout-Plugin');
-
-    this.hudSelectorButtons = hudSelectorButtons;
     this.sendAllProperties = this.sendAllProperties.bind(this);
+
+    this.hudPositionSelectors = hudSelectorRadios ? hudSelectorRadios.split(',') : [];
```

Comment:
It now dawns on me that we may need to rethink this logic throughout the container. Imagine this HTML:

```html
<input type="radio" value="top" class="radio" />
<input type="radio" value="bottom" class="radio" />
<input type="radio" value="left" class="radio" />
<input type="radio" value="right" class="radio" />
<input type="radio" value="top" class="radio" />
```

If I were to instantiate this plugin:
```javascript
new HUDPlugin('input.radio', {});
```

I have two `top` values, but I don't have a comma to separate them into separate groups. The selector would instead give me five elements all of which are valid. We should probably support that somehow, and avoid parsing those commas ourselves (since `querySelector` can do that for us).

This is, however, a larger conversation outside of the scope of this PR and is something we'll need to schedule some time to discuss 👍

Reply:
I'm splitting the selector manually because I want to keep the groups of radios separated. Since all of the set up and validation is done on each group as a whole. But we can definitely discuss behavior and implementation :)

Reply:
As per our conversation offline, we decided that we're going to hold off on any refactoring for `RadioGroup` for now.